main.py

```diff
@@ -26,7 +26,7 @@ def lambda_function(arguments_to_values, prettyprinted=False):
     if prettyprinted:
         raise NotImplementedError
     else:
-        return "(lambda " + (COMMA.join(arguments_to_values.keys())) + ": " + CONTINUATION + ")(" + (COMMA.join(arguments_to_values.values())) + ")"
+        return "(lambda " + COMMA.join(arguments_to_values.keys()).replace("%", "%%") + ": " + CONTINUATION + ")(" + COMMA.join(arguments_to_values.values()).replace("%", "%%") + ")"
```

Comment:
We're returning a format string containing a %s, so if anything in arguments_to_values includes %, we need to write %% to get a literal % after interpolation.